openfisca_core/tests/assets/parameters.yaml

```diff
@@ -0,0 +1,124 @@
+
+# Regular parameter values : meta + VALUES handling variation of its value in time
+- variable: smic_horaire_brut
```

Comment:
Yes it would be better as "name" or "code" or Mahdi's suggestions. But I wanted to insist on the fact that *most parameters are linked to a unique `class X(Variable)`